product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Postgres Distributed (PGD) presets on BigAnimal
+navTitle: PGD presets on BigAnimal
+deepToC: true
+---
+
+## Commit scope
+
+[Commit scopes](../../../pgd/latest/durability/commit-scopes/) in PGD is a set of rules that describes the behavior of the system as transactions are committed. Because they define how transactions are replicated across a distributed database, they impact consistency and reliability.
+
+The actual behavior depends on which kind of commit scope a commit scope's rule uses: [Group Commit](/../../../pgd/latest/durability/group-commit/), [Commit At Most Once](/../../../pgd/latest/durability/camo/), [Lag Control](/../../../pgd/latest/durability/lag-control/), [Synchronous Commit](/../../../pgd/latest/durability/synchronous_commit/), or a combination of these.
+
+Customizing these settings can be complex, presenting a challenge for users. 
+
+### BigAnimal's commit_scope preset options
+
+BigAnimal's commit scope presets simplify managing commit scopes in PGD. These presets balance customization with ease of use, allowing users to enjoy PGD's benefits without the complexity of manual configurations:
```

Comment:
This para reads more like marketing :) Better to present in brief the four options

BigAnimal's commit scope presets balance customization with ease of use and reduce the options down to four presets.

The presets include a rule for high consistency within a subgroup and a separate rule for lag control. There are then combinations of these rules for high consistency within a subgroup and lag control for other subgroups, and high coniositency within the subgroup and with one or more nodes of an other subgroups.

Something along those lines.

Reply:
Cool, got it.